mne/viz/_3d.py

```diff
@@ -537,6 +538,11 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
         .. versionadded:: 0.16
         .. versionchanged:: 1.0
            Defaults to ``'terrain'``.
+    sensor_colors: array-like | None
```

Comment:
```suggestion
    sensor_colors : array-like | None
```

This should fix the docstring tests

Reply:
Seems like all tests except the macOS M1 test passed now